demos/asp-net-core/Divergent.Sales.API.Host/Controllers/OrdersController.cs

```diff
@@ -41,8 +40,7 @@ public IEnumerable<dynamic> Get(int pageIndex, int pageSize)
                     .Take(pageSize)
                     .Select(o => new
                     {
-                        Number = o.Id,
-                        o.Id,
+                        OrderNumber = o.OrderNumber,
```

Comment:
Must/should/could we change

```C#
.OrderBy(o => o.Id) //required by SQLite EF
```

to use `OrderNumber`?

Reply:
fixed